integration-tests/utils/integration-helpers.js

```diff
@@ -198,22 +200,30 @@ const enableDataCatalog = async (page, testUrl, token) => {
   await dismissNotifications(page)
 }
 
-const performAnalysisTabSetup = async (page, token, testUrl, workspaceName) => {
-  await page.goto(testUrl)
-  await findText(page, 'View Workspaces')
-
-  await overrideConfig(page, { isAnalysisTabVisible: true })
+const clickNavChildAndLoad = async (page, tab) => {
+  await noSpinnersAfter(page, { action: () => click(page, navChild(tab)) })
+}
 
+const viewWorkspaceDashboard = async (page, token, workspaceName) => {
   await click(page, clickable({ textContains: 'View Workspaces' }))
   await signIntoTerra(page, token)
   await dismissNotifications(page)
+  await fillIn(page, input({ placeholder: 'SEARCH WORKSPACES' }), workspaceName)
   await noSpinnersAfter(page, { action: () => click(page, clickable({ textContains: workspaceName })) })
-  await noSpinnersAfter(page, { action: () => click(page, navChild('analyses')) })
+}
+
+const performAnalysisTabSetup = async (page, token, testUrl, workspaceName) => {
```

Comment:
FYI @jdcanas some I did some refactoring here to share the functionality of viewing the workspace dashboard.